chef_master/source/config_rb.rst

```diff
@@ -73,6 +80,11 @@ This configuration file has the following settings:
 
       data_bag_encrypt_version 2
 
+``fips``
+  Allows OpenSSL to enforce FIPS-validated security during the chef-client run. Set to ``true`` to enable FIPS-validated security.
+
+  Changed in Chef server 12.13 to expose FIPS runtime flag on RHEL.
```

Comment:
This was pulled in as part of a replication between the settings in the knife.rb documentation and those in config.rb, as the two were beginning to splinter, with knife.rb getting more updates. 

The "Changed in Chef server 12.13 to expose FIPS runtime flag on RHEL." line does seem like a non-sequitur here, though. I think it's meant to be a very roundabout way of saying that FIPS mode is only supported on RHEL (which is true but it also supports Oracle, CentOS and I think windows)...in other words that last sentence needs to be replaced.